gcp_variant_transforms/vcf_to_bq_preprocess.py

```diff
@@ -87,6 +90,24 @@ def _get_inferred_headers(variants,  # type: pvalue.PCollection
   return inferred_headers, merged_header
 
 
+# TODO(hanjohn): Add an e2e test
+def _estimate_disk_resources(p, input_pattern):
+  # type: (pvalue.PCollection, str) -> (pvalue.PCollection)
+  # TODO(hanjohn): Add support for `ReadAll` pattern for inputs with very large
+  # numbers of files.
+  result = (
+      p
+      | 'InputFilePattern' >> beam.Create([input_pattern])
+      | 'ReadFileSizeAndSampleVariants' >> vcf_file_size_io.EstimateVcfSize(
+          input_pattern, _SNIPPET_READ_SIZE)
```

Comment:
Hm I could imagine changing this eventually to a flag; I added a TODO to indicate that in the code. Would you prefer the change to be in the same commit though?

Reply:
Yeah, I think it's better to move it to vcf_file_size_io since general consensus is that we want to reduce the amount of flags if possible - I don't quite envision when clients would be fidgeting with this number. Seems like we should instead figure out an optimal amount (which 50 seems to be, especially given that difference between reading 1 line or 50 lines is negligible) and set it by default for all use cases.

Reply:
sg, thanks! done